upgrade/steps-helpers.go

```diff
@@ -359,18 +359,19 @@ func gitRefsOf(ctx context.Context, url, kind string) (gitRepoRefs, error) {
 		contract.Assertf(len(parts) == 2,
 			"expected git ls-remote to give '\t' separated values, found line %d: '%s'",
 			i, line)
-		refsToBranches[parts[0]] = parts[1]
+		branchesToRefs[parts[1]] = parts[0]
 	}
-	return gitRepoRefs{refsToBranches, kind}, nil
+	return gitRepoRefs{branchesToRefs, kind}, nil
+
 }
 
 type gitRepoRefs struct {
-	refsToLabel map[string]string
-	kind        string
+	LabelToRef map[string]string
```

Comment:
Why did we change capitalization here?
```suggestion
	labelToRef map[string]string
```

Reply:
No reason; I can flip it back.